lib/dotcom_web/components/trip_planner/trip_planner_results_section.ex

```diff
@@ -72,26 +72,27 @@ defmodule DotcomWeb.Components.TripPlanner.TripPlannerResultsSection do
               }}
          } = assigns
        ) do
-    %{itineraries: itineraries, summary: summary} = results |> Enum.at(itinerary_group_index)
+    with %{itineraries: itineraries, summary: summary} <-
+           results |> Enum.at(itinerary_group_index) do
```

Comment:
Is this change necessary? 

I only ask because Anthony is currently working on a refactor that will overhaul (possibly remove or rename?) this file, and this could generate git conflicts.

Reply:
I put it there because I ran into an error where this returned `nil` and failed the pattern match. hopefully the refactor will clearly obviate this!